package.json

```diff
@@ -27,34 +27,47 @@
   ],
   "scripts": {
     "build": "unbuild",
+    "prepare": "esno ./scripts/vendor.ts",
     "dev": "vitest dev",
     "dev:ui": "vitest dev --ui",
     "lint": "eslint --cache --ext .ts,.js,.mjs,.cjs . && prettier -c src test",
     "lint:fix": "eslint --cache --ext .ts,.js,.mjs,.cjs . --fix && prettier -c src test -w",
     "prepack": "pnpm run build",
     "typecheck": "tsc --noEmit",
     "release": "pnpm run test run && changelogen --release && npm publish && git push --follow-tags",
-    "test": "vitest"
+    "test": "vitest",
+    "test:build": "TEST_BUILD=true vitest",
+    "test:full": "pnpm run test && pnpm run build && pnpm run test:build"
   },
   "dependencies": {
     "@babel/parser": "^7.22.15",
     "@babel/types": "^7.22.15",
-    "recast": "^0.23.4"
+    "source-map-js": "^1.0.2"
```

Comment:
Shall we somehow mock and remove this too or it is essential dep?

Reply:
We do expose an `map` object, but I am not sure if anyone is using it. If you think that makes sense, we could introduce a breaking change and not to expose it any longer.

Reply:
A major version to allow saving as much as possible seems a nice idea! (BTW i will try to locally have a look before release if we can save more, it is a nice perf PR!)